x-pack/test/functional/apps/ml/settings/calendar_edit.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { asyncForEach } from './common';
+
+export default function ({ getService }: FtrProviderContext) {
+  const ml = getService('ml');
+  const esArchiver = getService('esArchiver');
+  const comboBox = getService('comboBox');
+
+  const calendarId = 'test_edit_calendar_id';
+  const testEvents = [
+    { description: 'event_1', start_time: 1513641600000, end_time: 1513728000000 },
+    { description: 'event_2', start_time: 1513814400000, end_time: 1513900800000 },
+  ];
+  const jobConfigs = [
+    ml.commonConfig.getADFqMultiMetricJobConfig('test_calendar_ad_1'),
+    ml.commonConfig.getADFqSingleMetricJobConfig('test_calendar_ad_2'),
+  ];
+  const newJobGroups = ['farequote'];
+
+  describe('calendar edit', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/farequote');
+      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+
+      await asyncForEach(jobConfigs, async (jobConfig) => {
+        await ml.api.createAnomalyDetectionJob(jobConfig);
+      });
+
+      await ml.api.createCalendar(calendarId, {
+        job_ids: jobConfigs.map((c) => c.job_id),
+        description: 'Test calendar',
+      });
+      await ml.api.createCalendarEvents(calendarId, testEvents);
+
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    afterEach(async () => {
+      await ml.api.deleteCalendar(calendarId);
+    });
+
+    it('calendar edit loads the calendars list and finds calendar to edit', async () => {
+      await ml.testExecution.logTestStep('calendar creation loads the calendar management page');
```

Comment:
This should be `calendar edit ...`.

Reply:
Updated here https://github.com/elastic/kibana/pull/77566/commits/20c6b2d06d4a964077e710a545e9fdc6286b2e5f